typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -520,35 +509,51 @@ class ContextFunder {
   }
 
   private async attemptToClaimFromIgp(chain: ChainName) {
-    const igpClaimThresholdEther = igpClaimThresholdPerChain[chain];
+    const igpClaimThresholdEther =
```

Comment:
nit: the intention here was not obvious for me on initial reading. 

Maybe something like 
```
let igpClaimThreshold = this.igpClaimThresholdPerChain[chain]
if (!igpClaimThreshold) {
   // set igpClaimThreshold to a 5th of the desiredBalancePerChain 
}
```

Reply:
it ends up making the code more verbose than it needs to imo, here it's kind of concise and to-the-point that if we don't have a config for a given chain we skip it

would rather just leave a comment for future readers

Reply:
e.g.
```ts
    const igpClaimThresholdEther = this.igpClaimThresholdPerChain[chain];
    let igpClaimThreshold: BigNumber;
    if (!igpClaimThresholdEther) {
      logger.warn(
        { chain },
        `No IGP claim threshold for chain ${chain}, inferring threshold from desired balance...`,
      );

      const desiredBalanceEther = this.desiredBalancePerChain[chain];
      if (!desiredBalanceEther) {
        logger.warn(
          { chain },
          `No desired balance for chain ${chain}, skipping IGP claim`,
        );
        return;
      }

      igpClaimThreshold = ethers.utils.parseEther(desiredBalanceEther).div(5);
    } else {
      igpClaimThreshold = ethers.utils.parseEther(igpClaimThresholdEther);
    }
```

Reply:
Comment for future readers is good :)